TheFourFifths/consus
# Bad input

### Summary

Sanitize the inputs server side

### Checklist


Finally, is it ready to be reviewed and merged: yes :white_check_mark: 

### How to Review

1. Pull this branch
2. On consus-client pull alternate-checkout story branch
3. Search for student with invalid character in student id (non-alphanumeric)
4. Network tab should show failure JSON that states, "Invalid characters in student ID"

NOTE: There is more functionality, but dev is required client side because otherwise the input would be sanitized before it gets to the server.
 

### Links

Put links to Jira tickets, and anything else, here.

- [TFF-90](https://msoese.atlassian.net/browse/TFF-90)
